### PR TITLE
Jc update filip

### PR DESCRIPTION
**Pull request by trhlikfilip**
Added Journal Club content

### CODE REVIEW

File: our-initiatives/journal-club/index.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import DocCardList from '@theme/DocCardList'
+
+# 📚 Journal Club
+
+The UCL Artificial Intelligence Society runs regular journal club sessions at which leading researchers in both academia and industry share the latest developments in the world of artificial intelligence and machine learning.
+
+## The Programme
+
+ - October 9th - [Yves-Alexandre de Montjoye](https://uclaisociety.co.uk/our-initiatives/journal-club/journal-club-1)
```
Comment (TheRootOf3) [inline our-initiatives/journal-club/index.mdx]: Please change this to a relative link, like below.
```suggestion
 - October 9th - [Yves-Alexandre de Montjoye](/our-initiatives/journal-club/journal-club-1)
```

File: our-initiatives/journal-club/index.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ - October 24th - Mihaela van der Schaar
+
+## Our Team
+
+This academic year, the tutorial series is being delivered by the following people:
+
+- [Filip Trhlik](#) (Head of Journal Club)
+- [Damien Bose](#) (Journal Club Officer)
```
Comment (TheRootOf3) [inline our-initiatives/journal-club/index.mdx]: If you have a personal website, feel more than free to link it here!

File: our-initiatives/journal-club/journal-club-1.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 2
+---
+
+# Yves-Alexandre de Montjoye: The search for anonymous data 
+
+**Date: 9th October 2023**
+
+![Yves-Alexandre de Montjoye](../../static/img/journal-1-image.jpeg)
```
Comment (TheRootOf3) [inline our-initiatives/journal-club/journal-club-1.md]: Please create a directory `jc` under `img` and change the path to the following.
```suggestion
![Yves-Alexandre de Montjoye](/img/jc/journal-1-image.jpeg)
```

File: our-initiatives/journal-club/journal-club-1.md
Comment (TheRootOf3) [inline our-initiatives/journal-club/journal-club-1.md:1]: nit: if you have time, you may want to add some styling/emojis here and there!

File: static/img/journal-1-image.jpeg
Comment (TheRootOf3) [inline static/img/journal-1-image.jpeg:1]: As mentioned earlier, please move under `static/img/jc/`.

**Review by TheRootOf3 — CHANGES_REQUESTED** [review]
Please review the comments, minor changes needed.

File: our-initiatives/journal-club/index.mdx (hunk 2 above)
Comment (trhlikfilip) [reply our-initiatives/journal-club/index.mdx]: hmm..I do but I might put my LinkedIn there

Comment (TheRootOf3) [reply our-initiatives/journal-club/index.mdx]: That works as well!

**Pull request closed without merging**